.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "toto/ssl_crypto"]
 	path = toto/ssl_crypto
-	url = git@github.com:/santiagotorres/ssl_crypto
+	url = https://github.com/secure-systems-lab/ssl_crypto
```

Comment:
For now we should just update the protocol. Once we have done all the necessary merges and tested the new additions of `secure-systems-lab/ssl_crypto` we can also change the address.